mender/values.yaml

```diff
@@ -69,6 +69,11 @@ default:
     enabled: false
     minAvailable: 1
 
+  # custom default envs for every service
+  customEnvs: []
```

Comment:
Can we `s/customEnvs/env/` to align it with Alvaldi?

Reply:
I tried, but I found two issues:
* The Alvaldi [utility](https://github.com/NorthernTechHQ/alvaldi-helm/blob/main/alvaldi/templates/_helpers.tpl#L261) is a bit different: it's not merging the envs (afaik) and I need a merge
* many keys `<service>.env` are already in place, with different notations:
  ```
    env:
      TENANTADM_MIDDLEWARE: prod
      TENANTADM_SERVER_PRIV_KEY_PATH: /etc/tenantadm/rsa/private.pem
  ```
But the Alvaldi notation implies:
```
env:
  - name: TENANTADM_MIDDLEWARE
    value: prod
  - name: TENANTADM_SERVER_PRIV_KEY_PATH
    value: /etc/tenantadm/rsa/private.pem
```

and this requires a delicate migration and a breaking change. Imho we should keep it simple with a key different than `env`, but let me know what you prefer.
If you want to align with Alvaldi, I'd prefer moving Alvaldi to `s/env/customEnvs/` or similar

Reply:
Ahh, ok. I was not aware.